charts/opentelemetry-operator/values.yaml

```diff
@@ -49,7 +49,7 @@ kubeRBACProxy:
       memory: 64Mi
 
 ## Admission webhooks make sure only requests with correctly formatted rules will get into the Operator.
-##
+## Disabling webhooks requires the ENABLE_WEBHOOKS env var to be set to false in order for the operator pod to start.
```

Comment:
Should this env var just be set in the helm template instead of asking users for setting it manually?

Reply:
@dmitryax The env var can be set trough the helm values [here](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-operator/values.yaml#L33)

Reply:
mm, not really sure. It doesn't feel natural to set env vars for users, but I can do it ...

Reply:
Do you see a use case when `ENABLE_WEBHOOKS` shouldn't be set to false while `Values.admissionWebhooks.enabled=false`?

Reply:
@dmitryax set the env var when both values set to false